src/app.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+import { Header } from './header';
+import { Main } from './main';
+import { Footer } from './footer';
+
+import { Error } from './error';
+
+import  "./app.scss";
+
+export const App = (
+	<div>
+		< Header />
+		< Error
+			text = "This is an error"
+			color = "red"
```

Comment:
there are shouldn't be spaces before and after the sign equal 
`text="This is an error"`